### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by aconitumnapellus**
##### SUMMARY
Bump version to newest tag

##### ISSUE TYPE
 - Bugfix Pull Request

### CODE REVIEW

**Review by s3lph — APPROVED** [review]

File: backupctl/version.py [hunk 1]
```diff
@@ -3,4 +3,4 @@
 
 """Version."""
 
-__version__ = "2.0.1"
+__version__ = "2.0.2"
```
Comment (s3lph) [inline backupctl/version.py:6]: Shouldn't this become 2.0.3 now? 2.0.2 is already released

**Review by s3lph — CHANGES_REQUESTED** [review]

File: backupctl/version.py (hunk 1 above)
Comment (aconitumnapellus) [reply backupctl/version.py:6]: I thought about bumping the version __after__ i create the 2.0.3 tag, but i can change that to 2.0.3 too.

Comment (s3lph) [reply backupctl/version.py:6]: Why would you want to do that? Then the version tagged 2.0.3 is would still report 2.0.2 in `--version`

Comment (aconitumnapellus) [reply backupctl/version.py:6]: True, ill change it to 2.0.3

**Review by s3lph — APPROVED** [review]

**Pull request merged**